package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@microsoft/azure-iot-ux-fluent-controls",
-    "version": "6.2.5",
+    "version": "6.3.0",
```

Comment:
This is a breaking change, so we should either increment major version or expose the non-portal component under a different name (e.g., just `Panel` to match Office Fabric's terminology)

Reply:
Discussed offline, we'll exposed one ContextPanel component with a portal props